python/cudf/cudf/pylibcudf_tests/test_quantiles.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import numpy as np
+import pyarrow as pa
+import pyarrow.compute as pc
+import pytest
+from utils import assert_column_eq, assert_table_eq
+
+import cudf._lib.pylibcudf as plc
+
+# Map pylibcudf interpolation options to pyarrow options
+interp_mapping = {
+    plc.types.Interpolation.LINEAR: "linear",
+    plc.types.Interpolation.LOWER: "lower",
+    plc.types.Interpolation.HIGHER: "higher",
+    plc.types.Interpolation.MIDPOINT: "midpoint",
+    plc.types.Interpolation.NEAREST: "nearest",
+}
+
+
+@pytest.fixture(scope="module", params=[[1, 2, 3, 4, 5], [5, 4, 3, 2, 1]])
+def pa_col_data(request, numeric_pa_type):
+    return pa.array(request.param, type=numeric_pa_type)
+
+
+@pytest.fixture(scope="module")
+def plc_col_data(pa_col_data):
+    return plc.interop.from_arrow(pa_col_data)
```

Comment:
Maybe combine these as we discussed in the other PR? Or you can wait for a follow-up, whatever you prefer.

Reply:
Thanks for the review, I'm planning on addressing in followup.